src/gui/redux/store.js

```diff
@@ -21,6 +22,7 @@ const store = createStore(
     }),
     model,
     router: router.reducer,
+    state,
```

Comment:
reducer `model` contains component `props`.
reducer `state` contains component `state`.

Should we combine them in one `component.props` and `component.state`